wbez/ferris-ds
# [sc-16908] Update SASS Version

## Updated:

- node version 14
- sass 1.62.0
- styleline 14.16.1

Added stylelint rules to ignore stricter changes for the updated version. 
- No complier issues
- No class or id changes

## Testing

1. nuke node_modules && package-lock.json
2. use node 14 or higher
3. npm i && npm dev
4. check that assests complied correctly
